integrations/symfony-bundle.rst

```diff
@@ -167,6 +147,39 @@ Authentication
                 plugins: ['httplug.plugin.authentication.my_wsse']
 
 
+List of services
+````````````````
+
++----------------------------------+---------------------------------------------------------------------+
+| Service id                       | Description                                                         |
++==================================+=====================================================================+
+| httplug.message_factory          | Service* that provides the `Http\Message\MessageFactory`            |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.uri_factory              | Service* that provides the `Http\Message\UriFactory`                |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.stream_factory           | Service* that provides the `Http\Message\StreamFactory`             |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client.[name]            | | This is your HttpClient that you have configured.                 |
+|                                  | | With the configuration below the name would be `acme_client`.     |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client                   | This is the first client configured or a client named `default`.    |
++----------------------------------+---------------------------------------------------------------------+
+| | httplug.plugin.content_length  | | These are plugins that are enabled by default.                    |
+| | httplug.plugin.decoder         | | These services are not public and may only be used when configure |
+| | httplug.plugin.error           | | HttpClients or other services.                                    |
```

Comment:
These services are not public and can only be used to configure

(its not may, private services are just not available after the container is build. "when configure" is not correct, it would be "when configuring", but "to configure" is active and thus easier to understand)